cmd/draft/connect.go

```diff
@@ -59,19 +72,43 @@ func (cn *connectCmd) run(runningEnvironment string) (err error) {
 		return err
 	}
 
-	connection, err := deployedApp.Connect(client, config, containerName)
+	var ports []string
+	if len(overridePorts) == 0 {
+		if deployedApp.OverridePorts != "" {
```

Comment:
Can we simplify the cyclical branch logic a little bit here?

```
if deployedApp.OverridePorts != "" {
	s := strings.Join(strings.Fields(deployedApp.OverridePorts), " ")
	ports = strings.Split(s, " ")
}

// --override-port takes precedence
if len(overridePorts) != 0 {
	ports = overridePorts
}
```

The end result should be the same, but the branch logic should be much simpler to grok.

Also see the below comment about making `OverridePorts` an array of strings instead of one long string. That would further simplify the code here as well as in draft.toml:

```
if len(deployedApp.OverridePorts) != 0 {
	ports = deployedApp.OverridePorts
}

// --override-port takes precedence
if len(overridePorts) != 0 {
	ports = overridePorts
}
```

and in draft.toml:

```
override-ports = ["8080:80", "8081:81"]
```